### ISSUE TITLE
Support Different Note Types

### ISSUE THREAD

**Issue by dumbledorian-polymath** [JeppeKlitgaard/ObsidianAnkiBridge#26]
**Is your feature request related to a problem? Please describe.**
First of all, thank you very much for the great plugin! Unless I am mistaken, I noticed that the plugin seems to only support the default note types in Anki (i.e. Basic and Cloze). A few days ago, I exported some cards to Anki using the plugin and updated some cards to have a custom note type (I have some simple modifications like Basic (+reverse)). When I synced with Anki again, it seems to have erased those cards and created new ones, and the spaced repetition has reset. I understand this is happening because the Obsidian Vault contains the "true" information. 

**Describe the solution you'd like**
I do like that Obsidian contains the "true" information, but it would be nice to create custom note cards (and basically not worry about having to modify cards in Anki once exported). Please let me know if I can help, since it seems so far that that the Basic note type is "baked in" using the codeblock, and a custom note type would require defining a flexible codeblock, which could be a lot of work. 

**Describe alternatives you've considered**
I think Obsidian_to_Anki plugin support custom note type, but I prefer this plugin for its cleanliness and rendering.

**Comment by JeppeKlitgaard** [comment]
Unfortunately I am exceedingly busy with other parts of life at the moment, but I do hope to support some basic custom note cards in the future.

The flexibility of Anki's custom notes does mean that making something that will magically fit everything won't be possible. For example, basic blueprints will inevitably be limited to note types that have two fields. Support for reversed basic notes should not be too hard.

The underlying implementation is already somewhat ready for custom note types, but there is a lot of programmatic plumbing to be done.

TL;DR: Will add feature in the future, but likely this is not going to be in the short/medium term.
